drivers/ethernet/phy_gecko.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2019 Interay Solutions B.V.
+ * Copyright (c) 2019 Oane Kingma
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* SiLabs Giant Gecko GG11 Ethernet PHY driver. */
+
+#include <errno.h>
+#include <kernel.h>
+#include <net/mii.h>
+#include "phy_gecko.h"
+
+#define LOG_MODULE_NAME eth_gecko_phy
+#define LOG_LEVEL CONFIG_ETHERNET_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
This could also shortened to
```
#include <logging/log.h>
LOG_MODULE_REGISTER(eth_gecko_phy, CONFIG_ETHERNET_LOG_LEVEL);
```
or you could just use the eth_gecko module too, in which case this could be written as
```
#include <logging/log.h>
LOG_MODULE_DECLARE(eth_gecko, CONFIG_ETHERNET_LOG_LEVEL);
```

Reply:
Changed to the first suggestion